sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+# Generated by ./bqetl generate events_daily
+
+---
+friendly_name: 'Google Ad Campaign Cost Breakdown'
+description: >
+  A breakdown of google ad campaign costs per click, install, and DOU.
+owners:
+  - ctroy@mozilla.com
+labels:
+  application: fenix
+  schedule: daily
+  incremental: true
+scheduling:
+  dag_name: bqetl_campaign_cost_breakdowns
+  referenced_tables: [
+    [
+      'moz-fx-data-bq-fivetran',
+      'google_ads',
+      'campaign_history'
+    ],
+    [
+      'moz-fx-data-shared-prod',
+      'fenix',
+      'marketing_attributable_metrics'
+    ]
+  ]
```

Comment:
```suggestion
  ]
  depends_on:
    - task_id: fenix_derived__attributable_clients__v1
      dag_name: bqetl_org_mozilla_firefox_derived
      execution_delta: 1h

```

Reply:
If you want this DAG to run **only** if the referenced tables have been updated, you may add the dependency and then run the command to update the DAG `./bqetl dag generate bqetl_campaign_cost_breakdowns`.

I.e. `moz-fx-data-shared-prod.fenix.marketing_attributable_metrics` ultimately depends on `wait_for_fenix_derived__attributable_clients__v1`.

Reply:
Thank you!!